conf/application.conf

```diff
@@ -45,7 +45,7 @@ play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoHmrcModule"
 
 play.modules.enabled += "uk.gov.hmrc.merchandiseinbaggage.config.MerchandiseInBaggageFrontendConfigModule"
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
```

Comment:
`localhost:12345` ? is this needed

Reply:
yeah, check this page for more info: https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=PLATUI&title=Integrating+with+tracking+consent+-+a+guide+for+service+teams